src/output_widget.cc

```diff
@@ -0,0 +1,73 @@
+#include "output_widget.h"
+
+#include <gtkmm/textview.h>
+
+#include <memory>
+
+namespace bb
+{
+    class OutputWidget::Impl
+    {
+    public:
+        Impl();
+        void set_line(int line, const std::string &text);
+        void create_empty_lines(int n);
+        void delete_line(int line);
+
+        Gtk::TextView tv;
+        Glib::RefPtr<Gtk::TextBuffer> buffer;
+    };
+}
+
+// Internal implementation
+
+bb::OutputWidget::Impl::Impl()
+{
+    tv.set_can_focus(false);
+    buffer = tv.get_buffer();
```

Comment:
This initialization should be in the initialization list of the constructor, and if you want to follow my previous suggestion you must use `.get()` method to get the raw pointer

Reply:
you're right! 0b43538